python/cucim/.pre-commit-config.yaml

```diff
@@ -1,20 +0,0 @@
-# To install the git pre-commit hook run:
```

Comment:
Was this meant to be dropped?

Reply:
This shouldn't exist as far as I understand. The only config that should matter is in the repo root.

Reply:
Interestingly it seems we are including this one in [`MANIFEST.in`]( https://github.com/rapidsai/cucim/blob/dbbeaf92f7f5175b0eb47bb88aa2e3b74c88a5cb/python/cucim/MANIFEST.in#L20 ) though

In any event, agree we should only have this in one place. If we stick to dropping this one, the `MANIFEST.in` needs an update

Reply:
Good catch! Removed from `MANIFEST.in` in 93c2c1ebbdb0e12768578eb851d2ef527227e774.

Reply:
Thanks Bradley! 🙏

Given the file is still here (just in another location), it is not really being dropped. If it turns out we need to move it for whatever reason, we can do that in a follow up.